cymetric/metrics.py

```diff
@@ -441,3 +441,58 @@ def timelist(info):
 
 del _tldeps, _tlschema
 
+# Quantity per GigaWattElectric in Inventory [kg/GWe]
+_invdeps = ['']
```

Comment:
you will need some here:
dependencies are the existing tables (or metrics) you rely on to build the new one

Reply:
so I am guessing, `Inventory`, `Power`... (I am not sure this is there exact names...)